README.md

```diff
@@ -1,5 +1,7 @@
 # Pulumi Command Provider (preview)
 
+[![Go Reference](https://pkg.go.dev/badge/github.com/pulumi/pulumi-command.svg)](https://pkg.go.dev/github.com/pulumi/pulumi-command)
```

Comment:
Hi @RobbieMcKinstry the link looks broken: https://pkg.go.dev/github.com/pulumi/pulumi-command

Any idea?

There is this https://pkg.go.dev/github.com/brandonkal/pulumi-command/sdk/go/command

Suspicious.

Reply:
Perhaps https://pkg.go.dev/github.com/pulumi/pulumi-command/sdk